python/src/NAIS_doc.i.in

```diff
@@ -3,7 +3,7 @@
 
 Parameters
 ----------
-event : :class:`~openturns.RandomVector`
+event : :class:`~openturns.ThresholdEvent`
```

Comment:
this was actually correct: a RandomVector whose implementation is a ThresholdEvent

Reply:
I think that the algorithm will not work if we indicate a RandomVector instead of a ThresholdEvent for the first attribute.

Reply:
it depends, for subset sampling for example, we can have other RV than a threshold one, namely IntersectionEvent or UnionEvent
if that's not the case here it should be changed at the c++ level too